docs/dev/setup.rst

```diff
@@ -522,7 +522,8 @@ above in Server Setup) and if you have configured
 with your TUM Online account.
 
 In case you encounter any problems regarding JavaScript heap memory leaks when executing ``npm run serve`` or any other scripts from ``package.json``,
-you can add a memory limit parameter (``--max_old_space_size=5120``) in the script.
+you can add a `memory limit parameter <https://nodejs.org/docs/latest-v16.x/api/cli.html#--max-old-space-sizesize-in-megabytes>`__
```

Comment:
Wouldn't that be set by the `.npmrc` config file? Maybe it would be better to direct the reader there.